packages/create-react-app/index.js

```diff
@@ -182,7 +182,25 @@ function run(root, appName, version, verbose, originalDirectory, template) {
 
   install(allDependencies, verbose, function(code, command, args) {
     if (code !== 0) {
-      console.error(chalk.cyan(command + ' ' + args.join(' ')) + ' failed');
+      console.log();
+      console.error('Aborting installation.', chalk.cyan(command + ' ' + args.join(' ')), 'has failed.');
+      // On 'exit' we will delete these files from target directory.
+      var knownGeneratedFiles = [
+        'package.json', 'npm-debug.log', 'yarn-error.log', 'node_modules'
```

Comment:
I don't think it's well known that `!!~(-1) === false`, I think that `!== -1` is more explicit. 😄
If it's this easy, I guess there's no issue. Sorry, I'm really tired and trying to overthink things.

We should probably only apply this to the log files though, not everything. The only scenario where this would malfunction is if someone made a `package.json.temp` or similar during install -- an edge case for sure ... but we still shouldn't delete the file[s].